joomla/joomla-cms
# [4.0] Split head to metas/styles/scripts

Pull Request for Issue # .

### Summary of Changes
Introduce 3 new renderers for styles, metas and scripts
Patch head for B/C
~~Render scripts at the bottom on the page for Atum and Aurora~~

~~About rendering the scripts at the bottom of the page:~~
- ~~Will break inline scripts: Well if dev's are NOT using the API for the inline scripts then their code will break, but hey use the freaking API and your code is guaranteed that it won't break...~~
- ~~Please understand javascript: you CAN'T manipulate the DOM if the DOM is NOT LOADED!~~

### Testing Instructions
Apply the patch and make sure that nothing breaks (or gives console errors, apart the error for the tooltips that comes from Bootstrap)
Use the old head tag and verify that one works as well e.g. admin login page

### Expected result



### Actual result



### Documentation Changes Required
